src/Symfony/Component/Messenger/TraceableMessageBus.php

```diff
@@ -27,7 +27,7 @@ public function __construct(MessageBusInterface $decoratedBus)
     /**
      * {@inheritdoc}
      */
-    public function dispatch($message): void
+    public function dispatch($message, string $key = null): void
```

Comment:
`$key` should be forwarded to the decorated class

Reply:
fixed thanks!